net-misc/youtube-viewer/youtube-viewer-3.1.9.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit gnome2-utils eutils perl-module vcs-snapshot
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="http://trizen.googlecode.com"
+SRC_URI="https://github.com/trizen/youtube-viewer/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+RDEPEND="
+	>=dev-lang/perl-5.16.0
+	dev-perl/Data-Dump
+	dev-perl/libwww-perl
+	|| ( media-video/mplayer[X,network]
+		media-video/mplayer2[X,network]
+		media-video/mpv[X] )
+	dev-perl/LWP-UserAgent-Cached
```

Comment:
this is optional https://github.com/trizen/youtube-viewer/blob/3.1.9/Build.PL#L68